src/services/codefixes/importFixes.ts

```diff
@@ -0,0 +1,474 @@
+﻿/* @internal */
+namespace ts.codefix {
+
+    type ImportCodeActionKind = "CodeChange" | "InsertingIntoExistingImport" | "NewImport";

```

Comment:
They are only used internally for sorting / filtering. So no need to expose them